docs/fuel-101/sequencer.mdx

```diff
@@ -0,0 +1,45 @@
+
+By efficiently ordering transactions, sequencers allow Layer 2 solutions to significantly reduce transaction processing times and costs. This efficiency gain is essential for scaling blockchain networks and improving their usability for a wider range of applications, making the trade-off an integral part of optimizing blockchain performance.
+
+## Shared Sequencer
+
+> A shared sequencer model involves multiple entities or nodes taking turns or sharing the responsibility of sequencing transactions.
+
+Fuel leverages a shared sequencer an Application Blockchain Interface (ABCI), integrated with the Cosmos network. Validators are responsible for
+
+1. Establishing the order of Fuel's blocks
+2. Verifying these optimistically ordered blocks through fraud proofing
+
+Any misbehaving actors producing bad blocks within the system are penalized (slashed) on Ethereum. The sequencer network uses the `secp256k1` cryptographic algorithm, which is compatible with Ethereum's infrastructure.
+
+### 1. Transaction Batching
+
+The shared sequencer network to collect a large number of transactions over a certain period. Instead of each transaction being submitted individually to the base chain, the sequencer batches these transactions into a single package. This approach significantly reduces the number of transactions that the base chain has to process, as a single batched transaction on the base chain can represent hundreds or thousands of individual transactions.
+
+### 2. Off-Chain Computation
+
+Shared sequencers perform much of the computational work off-chain. This includes executing the transactions in the batch, computing the new state, and only then submitting the relevant data to the base chain. Since the computational work is done off-chain, the base chain does not need to expend resources on executing these transactions, which reduces its workload and the amount of data it needs to process and store.
+
+### 3. Data Compression
+
+Before submitting transactions to the base chain, sequencers employs various data compression techniques to reduce the size of the transaction data. This involves encoding the data in a more efficient format and uses cryptographic techniques to represent the data more compactly.
+
+### 4. Fraud Proofs
+
+Fuel is an optimistic rollup, meaning its shared sequencer network submits a summary of transactions along with a commitment to the new state. Transactions are presumed valid by default, but there is a [dynamic challenge period](https://forum.fuel.network/t/why-wait-a-week-fast-finality-optimistic-rollups/4471) during which fraudulent transactions can be disputed. This significantly reduces the finality time of transactions from 7 days to just a few blocks. The Merkle root generated can serve as evidence of transaction order for a fraud-proof system. All sequencer network block headers are posted back to Ethereum for transparency and security.
+
+## Why Shared Sequencer?
```

Comment:
Why Use a Shared Sequencer?

Reply:
Some other reasons to add:

* Cost efficiency
* Interoperability - it can act as a standardizing agent on how transactions are ordered and processed across l2s
* Scalability of transactions